docs/kb/developer-resources/updating-expiration-on-licenses.md

```diff
@@ -0,0 +1,47 @@
++++
+date = "2017-03-13T00:00:00Z"
+lastmod = "2017-03-13T00:00:00Z"
+title = "Updating Expiration on Licenses"
+weight = "999999"
+categories = [ "Knowledgebase", "Developer Resources" ]
++++
+
+A good use case for using the VendorAPI is license maintenance. In this example we want to find all licenses that are expiring soon (<5 days) in the unstable channel and extend them by a month. We are using our unstable channel to ship new features to some of our customers and want to extend their access to the channel by a month while we hammer out some bugs in this release.
```

Comment:
Recommend that we have an example the beta channel. 

also "while we hammer out some bugs in this release" -> "while we get feedback on new features"